packages/gmp/package.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+class Gmp(CMakePackage):
+    """GMP Wrapper"""
+
+    homepage = "https://github.com/fdplacido/GMP"
+    git      = "https://github.com/fdplacido/GMP.git"
+    url      = "https://github.com/fdplacido/GMP/archive/plfernan/wrapper_improvements.tar.gz"
+
+    maintainers = ['fdplacido']
+
+    version('plfernan/wrapper_improvements', branch='plfernan/wrapper_improvements')
```

Comment:
You could call this version`develop`, which indicates for spack that it is the newest:
https://spack.readthedocs.io/en/latest/packaging_guide.html#version-comparison